deploy/terraform/modules/common_infrastructure/infrastructure.tf

```
@@ -226,3 +226,21 @@ resource "azurerm_storage_account" "storage-bootdiag" {
   account_tier              = "Standard"
   enable_https_traffic_only = var.options.enable_secure_transfer == "" ? true : var.options.enable_secure_transfer
 }
+
+
+# PROXIMITY PLACEMENT GROUP ===============================================================================================
+
+resource "azurerm_proximity_placement_group" "ppg" {
+  count               = var.infrastructure.ppg.is_existing ? 0 : 1
```

Should be able to handle when ppg is not defined in the input for backward supportability.
Same applies to the data type.
```suggestion
  count               = lookup(var.infrastructure, "ppg", false) != false ? (var.infrastructure.ppg.is_existing ? 0 : 1) : 0
```
The suggest is not tested. Please verify before use.